lib/lightning/accounts/user_notifier.ex

```diff
@@ -107,14 +107,15 @@ defmodule Lightning.Accounts.UserNotifier do
   """
   def deliver_project_addition_notification(user, project) do
     role = Projects.get_project_user_role(user, project) |> Atom.to_string()
-    url = LightningWeb.RouteHelpers.project_dashboard_url(project.id)
```

Comment:
@rorymckinley just noting here that I changed the url functions here across this file to use verified routes, so we don't need to:

- use the `RouteHelpers` module anymore
- provide an Endpoint to the `url` function, since the `~p` ast carries the endpoint on it.